### ISSUE TITLE
[Error2]:No such file or directory arabic_reshaper/__version__.py 

### ISSUE THREAD

**Issue by HamadAlAbsi** [mpcabd/python-arabic-reshaper#53]
arabic_reshaper working fine with linux but not with Android.
It show error with `__version__.py` file not found.
I traced the problem till i find the issue in the 
`__init__.py` in the line 
_**`exec(open(os.path.join(os.path.dirname(__file__), '__version__.py')).read())`**_

I did solution for it by replacing that line by
_**`__version__ = '2.1.0'`**_

And now it is working fine. So I like to share my solution with you . 
Thank you

**Comment by mpcabd** [comment]
On Android I believe it's an issue with packaging your app, could you shed some light on how you're packaging it? I think this is similar to #40

**Comment by HamadAlAbsi** [comment]
> On Android I believe it's an issue with packaging your app, could you shed some light on how you're packaging it? I think this is similar to #40

as I told you in the main post I solved it by only copying the content of the version file directly to the init.py file.
for the buildozer I didn't mention it in the requirements I include arabic_reshaper folder inside my project folder so it worked for me. 
I included screen shot of my buildozer it is my pleasure to correct me if I did something wrong.
![Screenshot from 2020-09-03 12-56-03](https://user-images.githubusercontent.com/18414026/92100918-0f682a80-ede5-11ea-8572-c42b7d0c661d.png)
![Screenshot from 2020-09-03 12-56-43](https://user-images.githubusercontent.com/18414026/92100921-1000c100-ede5-11ea-983c-bb064d6ced9b.png)
